Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install --upgrade --no-cache-dir black
+
+ENTRYPOINT /usr/local/bin/black --check --diff  .
```

Comment:
this causes an extra layer of shell indirection which will be slower since it has to start a shell first (it makes the entrypoint `sh -c '/usr/local/bin/black ...'` -- you're better to use the json syntax:

```dockerfile
ENTRYPOINT ["/usr/local/bin/black", "--check", "--diff", "."]
```